frontend/src/user/UserList.js

```diff
@@ -30,6 +31,10 @@ class UserList extends Component {
     render() {
         const  {users, isLoading} = this.state
 
+        if (users.email != 'admin@gmail.com' && localStorage.getItem("idUser") != null) {
```

Comment:
You don't have to check the users' email, but the email from the logged in user as you did in dropmenu